pkg/bucket/bucket.go

```diff
@@ -165,3 +189,39 @@ func RunList(cmd *cobra.Command, args []string) {
 	fmt.Print(table.String())
 	fmt.Printf("\n")
 }
+
+// CreateTieringStructure creates a tering structure for a given bucket
+func CreateTieringStructure(BucketClass nbv1.BucketClass, BucketName string) (string, error) {
```

Comment:
It's not the correct place for this function, if you need to find a common place for using this function in both provisioner and CLI.

Reply:
I think @BarakSason tried to move it to util and got some issues. Any specific reason not to put in bucket.go? 
It does related to buckets...

Reply:
I don't think that in provisioner.go (which handles OBCs) we should call a function in a file that handles bucket CLI

Reply:
@romayalon Where do you suggest moving it to?
Moving this to util.go resulted in a circular dependency

Reply:
After discussing with @dannyzaken we agreed that bucketcalss.go is the right place for this code